### PR TITLE
Fix RtlHardcoded lint warnings

### LINKED ISSUE #772: Fix RtlHardcoded lints
When you run ./gradlew lint, a report is generated at collect_app/build/reports/lint-results.html with potential problems with Collect. Find all with RtlHardcoded id and fix those.

@sjain281297 you've fixed #747 quickly so maybe you want to take this one as well?

### PR DESCRIPTION
**Pull request by championoverthinker**
@grzesiek2010 please review

### CODE REVIEW

**Comment by grzesiek2010** [comment]
@sjain281297 
Please edit lint.xml file as you did #750 

The second thing - please keep one order in all edited files eg.:
android:layout_alignLeft="@+id/element"
android:layout_alignStart="@+id/element"
android:layout_alignRight="@+id/element"
android:layout_alignEnd="@+id/element"

it's more legible.

**Comment by championoverthinker** [comment]
Ok I will do it

**Comment by championoverthinker** [comment]
@grzesiek2010 please check now

File: collect_app/src/main/res/layout/geopoint_layout.xml [hunk 1]
```diff
@@ -57,29 +57,32 @@
...
 			android:id="@+id/reload_location"
-			android:drawableRight="@drawable/ic_add_location_black_36dp"
+			android:drawableEnd="@drawable/ic_add_location_black_36dp"
 			android:padding="15dp"
 			android:layout_margin="5dp"
 			android:layout_width="wrap_content"
 			android:layout_height="match_parent"
-			android:layout_weight="1" />
+			android:layout_weight="1"
+			android:drawableRight="@drawable/ic_add_location_black_36dp" />
```
Comment (grzesiek2010) [inline collect_app/src/main/res/layout/geopoint_layout.xml]: Only one small thing. Please move this line below or above "android:drawableEnd" to keep the order like in other cases and everything else is looks ok.

**Comment by grzesiek2010** [comment]
@sjain281297 it looks ok to me. Thanks and watch out there are still lots of lints to fix so I'll create new issues and your work is welcome :)

**Pull request merged**